### PR TITLE
Deeper into react prep

### PR DESCRIPTION
**Pull request by hasparus**
We started from this branch on [Meetup #1](https://www.meetup.com/pl-PL/Chop-Chop-Academy-Web-Apps/events/266580470/).

### CODE REVIEW

File: app/src/components/TodoList.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+const Todo = ({ children, ...rest }) => {
+  const [focused, setFocused] = useState(false);
+  const [hovered, setHovered] = useState(false);
+
+  return ((
+    <button
+      onBlur={() => setFocused(false)}
+      onFocus={() => setFocused(true)}
+      onPointerOut={() => setHovered(false)}
+      onPointerEnter={() => setHovered(true)}
```
Comment (drlech) [inline app/src/components/TodoList.jsx:24]: To mozna chyba samym CSSem zrobic (`:hover`, `:focus`)?

Comment (hasparus) [reply app/src/components/TodoList.jsx:24]: Good pick. I can then transition with between "✅" and "⬜" with CSS.

**Review by hasparus — COMMENTED** [review]
Oh. We've actually removed this file in the next PR and the case with Todo.tsx is a bit more complex.

File: app/src/components/TodoList.jsx (hunk 1 above)
Comment (hasparus) [reply app/src/components/TodoList.jsx:24]: https://github.com/ChopChopOrg/academy/pull/4#pullrequestreview-327363649

**Pull request merged**